tests/xfstests/enable_kdump.pm

```diff
@@ -19,10 +19,11 @@ use utils 'zypper_call';
 use power_action_utils 'power_action';
 use kdump_utils;
 use testapi;
+use ipmi_backend_utils 'use_ssh_serial_console';
 
 sub run {
     my $self = shift;
-    select_console('root-console');
+    check_var('BACKEND', 'ipmi') ? use_ssh_serial_console : select_console 'root-console';
```

Comment:
Unfortunately, we have same issue in many-many places, so I'm fine to merge this change, but as you face this issue, please consider moving this to select_console, so in theory every test will work on baremetal setups. I tried that myself, but failed as there are parts which has to be executed once, so it requires more than 30 minutes =(

Reply:
This merits its own PR, I think.

Reply:
Absolutely, but I would try to prevent this in general as it increases efforts of the refactoring later. I will create a ticket for that, let's see if we will be able to deliver.